src/main/java/com/revature/model/Job.java

```diff
@@ -13,9 +11,11 @@
 import javax.persistence.JoinColumn;
 import javax.persistence.JoinTable;
 import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
-
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;

```

Comment:
Do not add imports if they are not used here.

Reply:
Changes are made as per suggestion